.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    # Check for updates to GitHub Actions every week
+    interval: "weekly"
+  ignore:
+  - dependency-name: "*"
+    update-types: 
+    - version-update:semver-patch
```

Comment:
the version format is `major.minor.patch`, so this will let the dependabot ignore the update of patch.

So it will ignore the verison of `rajatjindal/krew-release-bot` now, ( They are keep at 0.0.x)


/cc @RainbowMango

Reply:
I don't get why we need to ignore patch versions. 
I think as long as it is an official release that usually follows the semver version `major.minor.patch`.